cmd/ocm/create/user/cmd.go

```diff
@@ -0,0 +1,144 @@
+/*
+Copyright (c) 2020 Red Hat, Inc.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+  http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package user
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/openshift-online/ocm-cli/pkg/config"
+	cmv1 "github.com/openshift-online/ocm-sdk-go/clustersmgmt/v1"
+	"github.com/spf13/cobra"
+
+	c "github.com/openshift-online/ocm-cli/pkg/cluster"
+)
+
+var args struct {
+	clusterKey string
+	group      string
+}
+
+var Cmd = &cobra.Command{
+	Use:     "user",
+	Aliases: []string{"users"},
+	Short:   "Configure user access for cluster",
+	Long:    "Configure user access for cluster",
+	Example: `# Add a user to the dedicated-admins group
+  ocm create user myuser --cluster=mycluster --group=dedicated-admins`,
```

Comment:
maybe add an example of how to create multiple users

Reply:
This is marked as resolved, but I don't see the example